coins/bitcoin/common/src/main/scala/co/ledger/lama/bitcoin/common/clients/grpc/mocks/InterpreterClientMock.scala

```diff
@@ -63,14 +57,20 @@ class InterpreterClientMock extends InterpreterClient {
     IO.pure(0)
   }
 
-  def getLastBlocks(accountId: UUID): IO[GetLastBlocksResult] = {
-    val lastBlocks: List[Block] = savedTransactions(accountId)
-      .map(_.block)
+  def getLastBlocks(accountId: UUID): IO[List[BlockView]] = {
+    val lastBlocks: List[BlockView] = savedTransactions(accountId)
+      .collect { case TransactionView(_, _, _, _, _, _, _, Some(block), _) =>
+        BlockView(
+          block.hash,
+          block.height,
+          block.time
+        )
+      }
       .distinct
       .sortBy(_.height)
       .reverse
```

Comment:
Would it be more efficient to sort like this (we remove a total iteration on the list) ?
```suggestion
      .sortBy(-_.height)
```